jenkins/job-configs/global.yaml

```diff
@@ -110,3 +110,6 @@
 
         # Get golang into our PATH so we can run e2e.go
         export PATH="${{PATH}}:/usr/local/go/bin"
+
+    # repos to be pulled for soak-jobs
+    soak-repos: '--bare'
```

Comment:
Please do not do this.

I realize this is easier for today you, but this makes it substantially harder for future you, or future collaborator to understand how these files work.

If you're going to define a `{soak-repos}` value please make it local to `bootstrap-ci-soak.yaml`